iso15118/shared/security.py

```diff
@@ -1323,8 +1320,8 @@ def get_ocsp_url_for_certificate(certificate: Certificate) -> str:
 
 
 def all_certificates_from_chain(
-    certificate_chain: CertificateChainV2, root_cert: Optional[Certificate]
-) -> List[Certificate]:
+    certificate_chain: CertificateChainV2, root_cert: Optional[bytes]
+) -> List[bytes]:
```

Comment:
What's the reason to change away from the Certificate type here and elsewhere?

Reply:
the data type was wrong... the confusion was that there are two Certificate datatypes:
1. https://github.com/SwitchEV/iso15118/blob/master/iso15118/shared/messages/iso15118_2/datatypes.py#L53
2. https://github.com/SwitchEV/iso15118/blob/master/iso15118/shared/security.py#L34

The ISO15118 data type is of the type of the 1. which is bytes and not of the type from the Cryptography library, which was what was being used in the security.py file

Reply:
I see -- is it worth adding a ticket to rename the ISO 15118 type to, say, `CertificateBytes` (or doing this as a follow-up PR if it's simple)?

Reply:
yes, let's do it in another PR